DemoServer/AdditionalLanguages/go/Queries/filteringResultsBasics/metadata.json

```diff
@@ -40,7 +40,16 @@
                 "",
                 "* More filtering options are available - see related links.",
                 "",
-                "* Since no index is specified within the query, RavenDB will create a matching Auto-Index if there is no existing auto-index that can answer this query (see auto-indexes)."
+                "* Since no index is specified within the query, RavenDB will create a matching Auto-Index if there is no existing Auto-Index that can answer this query.",
+                "",
+                "* The index-fields that will be created in the Auto-Index depend on the filtering method used:<br>",
+                "  * `Where()`<br>",
+                "    The index-field created will contain terms with an exact match to the queried document-field.<br><br>",
```

Comment:
"will contain terms with an exact match to the queried document-field"
maybe slightly more precise formulation would be "exact value of the queried document-field"
because documents are processed, this field is read and its verbatim value is stored within index